packages/core-mobile/app/new/features/portfolio/collectibles/components/CollectibleScreen.tsx

```diff
@@ -0,0 +1,368 @@
+} from '../hooks/useCollectiblesFilterAndSort'
+import { CollectibleDetailsContent } from './CollectibleDetailsContent'
+import {
+  CARD_SIZE_SMALL,
+  CollectibleDetailsHeader,
+  SNAP_DISTANCE
+} from './CollectibleDetailsHeader'
+
+type CollectibleScreenRouteParams = {
+  localId?: string
+  initial?: CollectibleFilterAndSortInitialState
+}
+
+export const CollectibleScreen = ({
+  localId,
+  initial
+}: CollectibleScreenRouteParams): ReactNode => {
+  const {
+    theme: { colors }
+  } = useTheme()
+  const navigation = useNavigation()
+  const insets = useSafeAreaInsets()
+  const headerHeight = useHeaderHeight()
+
+  const { collectibles } = useCollectiblesContext()
+  const { filteredAndSorted } = useCollectiblesFilterAndSort(
+    collectibles,
+    initial
+  )
+
+  const [currentIndex, setCurrentIndex] = useState(
+    filteredAndSorted?.findIndex(
+      item => item.localId.toLowerCase() === localId?.toLowerCase()
+    ) ?? -1
+  )
+
+  const collectible = useMemo(
+    () => filteredAndSorted[currentIndex],
+    [currentIndex, filteredAndSorted]
+  )
+
+  const isFirst = currentIndex === 0
+  const isLast = currentIndex === filteredAndSorted.length - 1
+
+  const scrollY = useSharedValue(0)
+  const bounceValue = useSharedValue(0)
+  const scrollViewRef = useRef<Animated.ScrollView>(null)
+
+  const scrollViewHandler = useAnimatedScrollHandler(event => {
+    'worklet'
+    scrollY.value = event.contentOffset.y
+  })
+
+  const onSeeMore = (): void => {
+    'worklet'
+    bounceValue.value = withSequence(
+      withTiming(-40, ANIMATED.TIMING_CONFIG),
+      withTiming(0, ANIMATED.TIMING_CONFIG)
+    )
+  }
+
+  const onPrevious = useCallback((): void => {
+    if (isFirst) return
+    setCurrentIndex(currentIndex - 1)
+  }, [currentIndex, isFirst])
+
+  const onNext = useCallback((): void => {
+    if (isLast) return
+    setCurrentIndex(currentIndex + 1)
+  }, [currentIndex, isLast])
+
+  const headerRight = useCallback((): ReactNode => {
+    if (!collectible) return null
+
+    return (
+      <View style={{ flexDirection: 'row', paddingRight: HORIZONTAL_MARGIN }}>
+        <Pressable
+          disabled={isFirst}
+          style={{
+            height: '100%'
+          }}
+          onPress={onPrevious}>
+          <View
+            style={{
+              padding: 8,
+              height: '100%',
+              opacity: isFirst ? 0.4 : 1,
+              justifyContent: 'center',
+              alignItems: 'center',
+              transform: [{ rotate: '180deg' }]
+            }}>
+            <Icons.Custom.ArrowDown
+              color={colors.$textPrimary}
+              width={24}
+              height={24}
+            />
+          </View>
+        </Pressable>
+        <Pressable
+          style={{
+            height: '100%'
+          }}
+          hitSlop={{
+            top: 0,
+            right: HORIZONTAL_MARGIN,
+            bottom: 0,
+            left: 0
+          }}
+          disabled={isLast}
```

Comment:
when the list of attributes is long, it doesn't let me scroll all the way to see the rest of it
![Screenshot_20250328_003630_Core Internal](https://github.com/user-attachments/assets/b3edff89-ebce-437e-85ac-8415f9337d5a)